### ISSUE TITLE
Ammonite import fails

### ISSUE THREAD

**Issue by ayoub-benali** [scalameta/metals#3294]
**Describe the bug**

Metals fails to import ammonite scripts after clicking on the prompt. Bellow is the stacktrace that is logged

```
metals: 2021.11.16 15:08:52 ERROR Error getting Ammonite 2.3.8-124-2da846d2 (scala 2.13.7)
ammrunner.CoursierError: Error fetching Ammonite 2.3.8-124-2da846d2 for scala 2.13.7
	at ammrunner.AmmoniteFetcher.maybeResult$1(AmmoniteFetcher.scala:90)
	at ammrunner.AmmoniteFetcher.command(AmmoniteFetcher.scala:108)
	at scala.meta.internal.metals.ammonite.Ammonite.command(Ammonite.scala:161)
	at scala.meta.internal.metals.ammonite.Ammonite.$anonfun$start$4(Ammonite.scala:239)
	at scala.Option.map(Option.scala:230)
	at scala.meta.internal.metals.ammonite.Ammonite.start(Ammonite.scala:239)
	at scala.meta.internal.metals.MetalsLanguageServer.executeCommand(MetalsLanguageServer.scala:1798)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.request(GenericEndpoint.java:120)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleRequest(RemoteEndpoint.java:261)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:190)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: coursierapi.error.MultipleResolutionError: Error downloading com.lihaoyi:ammonite-compiler_2.13.7:2.3.8-124-2da846d2
  not found: /home/ayoub/.ivy2/local/com.lihaoyi/ammonite-compiler_2.13.7/2.3.8-124-2da846d2/ivys/ivy.xml
  not found: https://repo1.maven.org/maven2/com/lihaoyi/ammonite-compiler_2.13.7/2.3.8-124-2da846d2/ammonite-compiler_2.13.7-2.3.8-124-2da846d2.pom
Error downloading com.lihaoyi:ammonite-repl-api_2.13.7:2.3.8-124-2da846d2
  not found: /home/ayoub/.ivy2/local/com.lihaoyi/ammonite-repl-api_2.13.7/2.3.8-124-2da846d2/ivys/ivy.xml
  not found: https://repo1.maven.org/maven2/com/lihaoyi/ammonite-repl-api_2.13.7/2.3.8-124-2da846d2/ammonite-repl-api_2.13.7-2.3.8-124-2da846d2.pom

	at coursierapi.error.MultipleResolutionError.of(MultipleResolutionError.java:28)
	at coursierapi.shaded.coursier.internal.api.ApiHelper$.doFetch(ApiHelper.scala:356)
	at coursierapi.shaded.coursier.internal.api.ApiHelper.doFetch(ApiHelper.scala)
	at coursierapi.Fetch.fetchResult(Fetch.java:244)
	at ammrunner.AmmoniteFetcher.maybeResult$1(AmmoniteFetcher.scala:87)
	at ammrunner.AmmoniteFetcher.command(AmmoniteFetcher.scala:108)
	at scala.meta.internal.metals.ammonite.Ammonite.command(Ammonite.scala:161)
	at scala.meta.internal.metals.ammonite.Ammonite.$anonfun$start$4(Ammonite.scala:239)
	at scala.Option.map(Option.scala:230)
	at scala.meta.internal.metals.ammonite.Ammonite.start(Ammonite.scala:239)
	at scala.meta.internal.metals.MetalsLanguageServer.executeCommand(MetalsLanguageServer.scala:1798)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.request(GenericEndpoint.java:120)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleRequest(RemoteEndpoint.java:261)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:190)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

:: <-  metals window/logMessage: {'message': '2021.11.16 15:08:52 ERROR Error getting Ammonite 2.3.8-124-2da846d2 (scala 2.13.7)\nammrunner.CoursierError: Error fetching Ammonite 2.3.8-124-2da846d2 for scala 2.13.7\n\tat ammrunner.AmmoniteFetcher.maybeResult$1(AmmoniteFetcher.scala:90)\n\tat ammrunner.AmmoniteFetcher.command(AmmoniteFetcher.scala:108)\n\tat scala.meta.internal.metals.ammonite.Ammonite.command(Ammonite.scala:161)\n\tat scala.meta.internal.metals.ammonite.Ammonite.$anonfun$start$4(Ammonite.scala:239)\n\tat scala.Option.map(Option.scala:230)\n\tat scala.meta.internal.metals.ammonite.Ammonite.start(Ammonite.scala:239)\n\tat scala.meta.internal.metals.MetalsLanguageServer.executeCommand(MetalsLanguageServer.scala:1798)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)\n\tat org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.request(GenericEndpoint.java:120)\n\tat org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleRequest(RemoteEndpoint.java:261)\n\tat org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:190)\n\tat org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)\n\tat org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)\n\tat org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)\n\tat java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:266)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)\n\tat java.lang.Thread.run(Thread.java:748)\nCaused by: coursierapi.error.MultipleResolutionError: Error downloading com.lihaoyi:ammonite-compiler_2.13.7:2.3.8-124-2da846d2\n  not found: /home/ayoub/.ivy2/local/com.lihaoyi/ammonite-compiler_2.13.7/2.3.8-124-2da846d2/ivys/ivy.xml\n  not found: https://repo1.maven.org/maven2/com/lihaoyi/ammonite-compiler_2.13.7/2.3.8-124-2da846d2/ammonite-compiler_2.13.7-2.3.8-124-2da846d2.pom\nError downloading com.lihaoyi:ammonite-repl-api_2.13.7:2.3.8-124-2da846d2\n  not found: /home/ayoub/.ivy2/local/com.lihaoyi/ammonite-repl-api_2.13.7/2.3.8-124-2da846d2/ivys/ivy.xml\n  not found: https://repo1.maven.org/maven2/com/lihaoyi/ammonite-repl-api_2.13.7/2.3.8-124-2da846d2/ammonite-repl-api_2.13.7-2.3.8-124-2da846d2.pom\n\n\tat coursierapi.error.MultipleResolutionError.of(MultipleResolutionError.java:28)\n\tat coursierapi.shaded.coursier.internal.api.ApiHelper$.doFetch(ApiHelper.scala:356)\n\tat coursierapi.shaded.coursier.internal.api.ApiHelper.doFetch(ApiHelper.scala)\n\tat coursierapi.Fetch.fetchResult(Fetch.java:244)\n\tat ammrunner.AmmoniteFetcher.maybeResult$1(AmmoniteFetcher.scala:87)\n\tat ammrunner.AmmoniteFetcher.command(AmmoniteFetcher.scala:108)\n\tat scala.meta.internal.metals.ammonite.Ammonite.command(Ammonite.scala:161)\n\tat scala.meta.internal.metals.ammonite.Ammonite.$anonfun$start$4(Ammonite.scala:239)\n\tat scala.Option.map(Option.scala:230)\n\tat scala.meta.internal.metals.ammonite.Ammonite.start(Ammonite.scala:239)\n\tat scala.meta.internal.metals.MetalsLanguageServer.executeCommand(MetalsLanguageServer.scala:1798)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)\n\tat org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.request(GenericEndpoint.java:120)\n\tat org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleRequest(RemoteEndpoint.java:261)\n\tat org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:190)\n\tat org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)\n\tat org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)\n\tat org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)\n\tat java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:266)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)\n\tat java.lang.Thread.run(Thread.java:748)\n', 'type': 4}
:: <-  metals window/showMessage: {'message': 'Error fetching Ammonite 2.3.8-124-2da846d2 for scala 2.13.7', 'type': 1}
:: <<< metals 19: {}
```

Strangely adding `// ammonite 2.4.1` to the top of the file fixed the problem. This version matches with my local ammonite version, should that matter ? 

**To Reproduce**

1. create any `*.sc` file 
2. Click on 'import' from the metals prompt
3. See error

**Expected behavior**
Ammonite build server to start correctly

**Installation:**

- Operating system: Linux
- Editor: Sublime
- Metals version: 0.10.9+42-07a66ff1-SNAPSHOT

**Search terms**

ammonite

**Comment by tgodzik** [comment]
Thanks for reporting! Ammonite scripts are not yet supported for 2.13.7, https://github.com/scalameta/metals/pull/3292 should fix that, but some errors popped up there,which I wasn't yet able to investigate.

**Comment by ayoub-benali** [comment]
So setting the version in the script forces Metal to download a more recent version of ammonite than the default one ?

**Comment by ckipp01** [comment]
> So setting the version in the script forces Metal to download a more recent version of ammonite than the default one ?

Correct, we use https://github.com/alexarchambault/ammonite-runner under the hood. The reason yours was failing was because that version wasn't published for 2.13.7, but putting your `// ammonite 2.4.1` on top caused that version of ammonite to be used.

**Comment by tgodzik** [comment]
This is now fixed in https://github.com/scalameta/metals/pull/3303 

Previously, we would revert to latest Scala 2.13 version in Ammonite for `automatic` version and we should revert to the latest ammonite supported version.
